core/java/iesi-core/src/main/java/io/metadew/iesi/openapi/ConnectionParser.java

```diff
@@ -0,0 +1,77 @@
+package io.metadew.iesi.openapi;
+
+import io.metadew.iesi.metadata.definition.connection.Connection;
+import io.metadew.iesi.metadata.definition.connection.ConnectionParameter;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.servers.Server;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ConnectionParser {
+    private static ConnectionParser INSTANCE;
+    public static String[] fakeEnvs = new String[] {"TST", "PRD", "ENVV"};
+
+
+    private ConnectionParser() {}
+
+    public synchronized static ConnectionParser getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new ConnectionParser();
+        }
+        return INSTANCE;
+    }
+
+    public  List<Connection> parse(OpenAPI openAPI) {
+        String name = openAPI.getInfo().getTitle();
+        String description = openAPI.getInfo().getDescription();
+        List<URL> adresses = getAdresses(openAPI.getServers());
+        return adresses.stream().map(address -> {
+            String env = fakeEnvs[adresses.indexOf(address)];
+
+            ConnectionParameter host = new ConnectionParameter(name, env, "host", getHost(address));
+            ConnectionParameter port = new ConnectionParameter(name, env, "port", getPort(address));
```

Comment:
this is an optional parameter. It should first be checked if its needed to add the parameters